t/run/switch-I-and-M.t

```diff
@@ -0,0 +1,49 @@
+#!./perl
+
+BEGIN {
+    chdir 't' if -d 't';
+    @INC = '../lib';
+}
+
+use strict;
```

Comment:
Is there a particular reason for not using `use warnings`, too?

Reply:
Most of the test files in `t/run/` pre-date the point in time where we became more obsessive about `use warnings;`.
```
$ ack -L 'use warnings;' t/run/*.t | wc -l
19
$ ack -l 'use warnings;' t/run/*.t | wc -l
8
```
I tried this file with `use warnings;`; none were emitted.  If we want all the files in `t/run/*.t` to have `use warnings;`, then I think we should create a separate issue/ and solicit a pull request for that.  (Good project for P5P beginner!)  If @DrHyde wants to add `use warnings;` here, I would have no objection.  But I think the p.r. is okay without it.

Reply:
Because it's missing from `switchM.t`, which is what this new test script was based on. I'll add it to both.